prepare_zephir_records/cid_minter/zephir_cluster_lookup.py

```diff
@@ -298,26 +300,28 @@ def formatting_cid_id_clusters(cid_id_list, other_id):
     return cid_ids_dict
 
 def valid_sql_in_clause_str(input_str):
-    """Validates if input is comma separated, single quoted strings with only digits.
+    """Validates if input is comma separated, single quoted strings.
 
     Returns:
         True: valid
         False: Invalid
 
         For example:
         True: 
-            "'1', '2345'"
             "'1'"
+            "'1', '2345'"
+            "'a'"
+            "'abc', 'xyz'"
         False: 
             "1, 2345"
-            "'abc', 'xyz'"
             ""
+            " "
     """
 
     if not input_str:
         return False
 
-    if re.search(r"^'(\d+)'(\s)*((\s)*(,)(\s)*('(\d+)'))*$", input_str):
+    if re.search(r"^(\s)*'(.+)'(\s)*((\s)*(,)(\s)*('(.+)'))*$", input_str):
```

Comment:
It it okay that ' ' is a valid input string (single quotes surrounding whitespace)?

Reply:
Ignore previous comment, this behavior matches the purpose of the function.

Reply:
Thanks Raiden for reviewing. This is to prepare a sting for the MySQL IN clause. When it is '' we do want to perform a select query.  Jing